spaces/S000175/properties/P000079.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000175
+property: P000079
+value: true
+refs:
+  - doi: 10.2307/2315929
+    name: Problem 5468 of The American Mathematical Monthly
+---
+
+Follows as the space is the quotient of first-countable spaces (Euclidean lines).
```

Comment:
Usually a quotient space is with respect to a single quotient map.  We know what we mean here, but do you think we should say something like quotient of a topological sum of ... ?